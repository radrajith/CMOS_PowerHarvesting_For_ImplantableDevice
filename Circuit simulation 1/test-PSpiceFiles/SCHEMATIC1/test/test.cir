** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../test-pspicefiles/test.lib" 
* From [PSPICE NETLIST] section of C:\Users\radra\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V2 0 5 .1 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
