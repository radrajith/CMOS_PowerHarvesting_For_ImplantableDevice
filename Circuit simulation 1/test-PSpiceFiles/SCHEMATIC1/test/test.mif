 lib="c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\test.lib", offset=43, size=44
 lib=C:\Cadence\SPB_17.2\tools\PSpice\Library\harris.lib, offset=241675, size=2147
