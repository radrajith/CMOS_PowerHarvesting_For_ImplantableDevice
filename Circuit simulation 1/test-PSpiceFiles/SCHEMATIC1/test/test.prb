[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS DC_SWEEP
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT x
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
BEGIN TRACE V(M1:g)
MARKERID 17
TRACEADDEXT
END TRACE V(M1:g)
BEGIN TRACE I(M1:d)
MARKERID 18
TRACEADDEXT
END TRACE I(M1:d)
BEGIN TRACE V(V2:+)
MARKERID 19
TRACEADDEXT
END TRACE V(V2:+)
BEGIN TRACE V(R2:2)
MARKERID 1
TRACEADDEXT
END TRACE V(R2:2)
BEGIN TRACE V(M1:g)
MARKERID 1
TRACEADDEXT
END TRACE V(M1:g)
BEGIN TRACE V(M3:g)
MARKERID 1
TRACEADDEXT
END TRACE V(M3:g)
BEGIN TRACE V(M2:g)
MARKERID 2
TRACEADDEXT
END TRACE V(M2:g)
BEGIN TRACE V(M3:g)
MARKERID 1
TRACEADDEXT
END TRACE V(M3:g)
BEGIN TRACE V(M2:g)
MARKERID 2
TRACEADDEXT
END TRACE V(M2:g)
BEGIN TRACE I(M1:d)
MARKERID 3
TRACEADDEXT
END TRACE I(M1:d)
BEGIN TRACE V(M2:s)
MARKERID 1
TRACEADDEXT
END TRACE V(M2:s)
BEGIN TRACE V(V2:+)
MARKERID 2
TRACEADDEXT
END TRACE V(V2:+)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
