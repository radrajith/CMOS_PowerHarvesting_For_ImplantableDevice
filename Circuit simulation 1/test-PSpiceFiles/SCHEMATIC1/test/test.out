
**** 10/29/16 13:55:24 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\t


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
.LIB "../../../test-pspicefiles/test.lib" 
* From [PSPICE NETLIST] section of C:\Users\radra\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V2 0 5 .1 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source TEST
X_M1         N08401 N06220 0 RFP30N06LE/HA
V_V1         N03452 0 5
V_V2         N05906 0 0
X_M2         N03452 N05906 N06220 RFP30N06LE/HA
D_D3         N03452 N08401 Dbreak 
R_R2         0 N06220  1000k TC=0,0 
X_M3         N07081 N05906 0 RFP30N06LE/HA
D_D4         N03452 N07081 Dbreak 
C_C1         0 N07081  1n  TC=0,0 
R_R3         N07081 N03452  100k TC=0,0 
C_C2         N08401 0  10u  TC=0,0 

**** RESUMING test.cir ****
.END

**** 10/29/16 13:55:24 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\t


 ****     Diode MODEL PARAMETERS


******************************************************************************




               Dbreak          X_M1.DBDMOD     X_M1.DBKMOD     X_M1.DESD1MOD   
          IS   10.000000E-15  380.000000E-15   10.000000E-15   10.000000E-15 
          BV                                                   13.54         
          RS     .34             .0112           .182          45.5          
          TT                   38.400000E-09                                 
         CJO  100.000000E-15    1.050000E-09                                 
        TRS1                    1.610000E-03    7.500000E-03                 
        TRS2                    6.080000E-06  -40.000000E-06                 


               X_M1.DESD2MOD   X_M1.DPLCAPMOD  X_M2.DBDMOD     X_M2.DBKMOD     
          IS   10.000000E-15    1.000000E-30  380.000000E-15   10.000000E-15 
           N                   10                                            
          BV   11.46                                                         
          RS                                     .0112           .182        
          TT                                   38.400000E-09                 
         CJO                  591.000000E-12    1.050000E-09                 
        TBV1 -757.600000E-06                                                 
        TBV2   -3.000000E-06                                                 
        TRS1                                    1.610000E-03    7.500000E-03 
        TRS2                                    6.080000E-06  -40.000000E-06 


               X_M2.DESD1MOD   X_M2.DESD2MOD   X_M2.DPLCAPMOD  X_M3.DBDMOD     
          IS   10.000000E-15   10.000000E-15    1.000000E-30  380.000000E-15 
           N                                   10                            
          BV   13.54           11.46                                         
          RS   45.5                                              .0112       
          TT                                                   38.400000E-09 
         CJO                                  591.000000E-12    1.050000E-09 
        TBV1                 -757.600000E-06                                 
        TBV2                   -3.000000E-06                                 
        TRS1                                                    1.610000E-03 
        TRS2                                                    6.080000E-06 


               X_M3.DBKMOD     X_M3.DESD1MOD   X_M3.DESD2MOD   X_M3.DPLCAPMOD  
          IS   10.000000E-15   10.000000E-15   10.000000E-15    1.000000E-30 
           N                                                   10            
          BV                   13.54           11.46                         
          RS     .182          45.5                                          
         CJO                                                  591.000000E-12 
        TBV1                                 -757.600000E-06                 
        TBV2                                   -3.000000E-06                 
        TRS1    7.500000E-03                                                 
        TRS2  -40.000000E-06                                                 


**** 10/29/16 13:55:24 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\t


 ****     MOSFET MODEL PARAMETERS


******************************************************************************




               X_M1.MOSMOD     X_M2.MOSMOD     X_M3.MOSMOD     
               NMOS            NMOS            NMOS            
       LEVEL    1               1               1            
           L    1.000000E-06    1.000000E-06    1.000000E-06 
           W    1.000000E-06    1.000000E-06    1.000000E-06 
         VTO    1.94            1.94            1.94         
          KP  139.2           139.2           139.2          
       GAMMA    0               0               0            
         PHI     .6              .6              .6          
      LAMBDA    0               0               0            
          IS    1.000000E-30    1.000000E-30    1.000000E-30 
          JS    0               0               0            
           N   10              10              10            
          PB     .8              .8              .8          
        PBSW     .8              .8              .8          
          CJ    0               0               0            
        CJSW    0               0               0            
        CGSO    0               0               0            
        CGDO    0               0               0            
        CGBO    0               0               0            
         TOX    1               1               1            
          XJ    0               0               0            
       UCRIT   10.000000E+03   10.000000E+03   10.000000E+03 
      DIOMOD    1               1               1            
         VFB    0               0               0            
        LETA    0               0               0            
        WETA    0               0               0            
          U0    0               0               0            
        TEMP    0               0               0            
         VDD    5               5               5            
       XPART    0               0               0            


**** 10/29/16 13:55:24 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\t


 ****     Resistor MODEL PARAMETERS


******************************************************************************




               X_M1.RBKMOD     X_M1.RDSMOD     X_M1.RSLVCMOD   X_M1.RVTOMOD    
           R    1               1               1               1            
         TC1    1.070000E-03    5.380000E-03    1.750000E-03   -2.150000E-03 
         TC2 -303.000000E-09   16.400000E-06    3.900000E-06   -5.430000E-06 


               X_M2.RBKMOD     X_M2.RDSMOD     X_M2.RSLVCMOD   X_M2.RVTOMOD    
           R    1               1               1               1            
         TC1    1.070000E-03    5.380000E-03    1.750000E-03   -2.150000E-03 
         TC2 -303.000000E-09   16.400000E-06    3.900000E-06   -5.430000E-06 


               X_M3.RBKMOD     X_M3.RDSMOD     X_M3.RSLVCMOD   X_M3.RVTOMOD    
           R    1               1               1               1            
         TC1    1.070000E-03    5.380000E-03    1.750000E-03   -2.150000E-03 
         TC2 -303.000000E-09   16.400000E-06    3.900000E-06   -5.430000E-06 


**** 10/29/16 13:55:24 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\t


 ****     Voltage Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_M1.S1AMOD     X_M1.S1BMOD     X_M1.S2AMOD     X_M1.S2BMOD     
         RON   10.000000E-06   10.000000E-06   10.000000E-06   10.000000E-06 
        ROFF     .1              .1              .1              .1          
         VON   -4.05           -1.5            -2.2             2.8          
        VOFF   -1.5            -4.05            2.8            -2.2          


               X_M2.S1AMOD     X_M2.S1BMOD     X_M2.S2AMOD     X_M2.S2BMOD     
         RON   10.000000E-06   10.000000E-06   10.000000E-06   10.000000E-06 
        ROFF     .1              .1              .1              .1          
         VON   -4.05           -1.5            -2.2             2.8          
        VOFF   -1.5            -4.05            2.8            -2.2          


               X_M3.S1AMOD     X_M3.S1BMOD     X_M3.S2AMOD     X_M3.S2BMOD     
         RON   10.000000E-06   10.000000E-06   10.000000E-06   10.000000E-06 
        ROFF     .1              .1              .1              .1          
         VON   -4.05           -1.5            -2.2             2.8          
        VOFF   -1.5            -4.05            2.8            -2.2          


Starting pseudo-transient algorithm.

WARNING(ORPSIM-16534): Using high values of ITL4 for Speed Level > 0 may increase total simulation job time.


          JOB CONCLUDED

**** 10/29/16 13:55:24 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-test"  [ c:\users\radra\box sync\senior fall 2016\ese 440\circuit simulation 1\test-pspicefiles\schematic1\t


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .23
